<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Resume</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Haowei Ni</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="resume.html">Resume</a>
</li>
<li>
  <a href="code.html">Plotly</a>
</li>
<li>
  <a href="mailto:&lt;hn2339@cumc.columbia.edu&gt;">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/&lt;nihaowei&gt;/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Resume</h1>

</div>


<p>400 West 113th St Apt 728</p>
<p>Cell: 6198870013</p>
<div id="education" class="section level2">
<h2>EDUCATION</h2>
<ul>
<li><p><strong>Columbia University, Mailman School of Public Health,</strong> New York, NY<br />
<br /> <right> Master of Science, Biostatistics Theory and Methods Track, expected May 2020 Relevant Courses: Probability, Data Science Computing With R, Epidemiology, Biostatistics Methods</p></li>
<li><p><strong>University of California San Diego,</strong> San Diego, CA<br />
<br /> B.S. in Biochemistry and Cell Biology, July 2018<br />
Provost Honors in Spring 2015, Winter 2015, Spring 2017, Winter 2017</p></li>
</ul>
</div>
<div id="work-experience" class="section level2">
<h2>WORK EXPERIENCE</h2>
<p><strong>Jiangsu Center for Disease Control and Prevention,</strong> Nanjing, China, Jun 2017-Sep 2017 <br /><strong>Lab Assistant and Data Analyst</strong></p>
<p>Project Investigation on the post-exposure vaccine treatment of Rabies in Jiangsu province</p>
<ul>
<li><p>Collected clinical and demographic information from 5295 patients who received Rabies Vaccine in Yancheng Municipal Hospital from 2016 to 2017.</p></li>
<li><p>Input data with Epidata and used Excel to preprocess the data set.</p></li>
<li><p>Conducted logistic regression analysis by using SPSS 18.0.</p></li>
<li><p>Applied MATLAB to perform Chi-seq test and to improve the analysis efficiency.</p></li>
<li><p>Focused on chronic disease epidemiology and acquired the fundamentals of the vaccine application.</p></li>
<li><p>Compared the clinical effectiveness of 2-1-1 vaccination method and 5 needle vaccination method in different age groups, providing data support for the local government and hospitals to optimize the treatment procedure of 2-1-1 method.</p></li>
</ul>
</div>
<div id="research-experience" class="section level2">
<h2>RESEARCH EXPERIENCE</h2>
<p><strong>The Scripps Research Institute,</strong> La Jolla, CA, Jan 2017- Sep 2017</p>
<p><strong>The Zhang Laboratory, Research Assistant</strong></p>
<ul>
<li><p>Proficient in MATLAB programming for experiment control over TIRFM and pressure-actuated mechanism.</p></li>
<li><p>Experienced in writing code for sequencing automation to optimize efficiency of data collection.</p></li>
<li><p>Strengthened independent experimental design skills and accurately followed research protocols.</p></li>
<li><p>Used RT-PCR to detect and amplify protein coding sequence and product.</p></li>
<li><p>Performed CHIP-seq to view histone modification and CRISPR/Cas9 to resolve the function of specific genes.</p></li>
</ul>
<p><strong>School of Life Sciences, Nanjing University,</strong> Jiangsu, China Jul 2016 â€“ Oct 2016 <strong>Biomedical Laboratory, Research Intern</strong></p>
<ul>
<li><p>Participated in the project New drug delivery systems and viable non-viral cationic polymers for gene therapy.</p></li>
<li><p>Worked in a team with other researchers as well as the primary investigator.</p></li>
<li><p>Used ApE file to find the sequence of the plasmid and learned the theory behind the Western Blotting experiment.</p></li>
<li><p>Improved lab skills like PCR and troubleshooting.</p></li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>SKILLS</h2>
<p><strong>Strong knowledge of:</strong></p>
<p>R studio, JAVA, MATLAB, SPSS18.0, SAS, Epidata <strong>Familiar with:</strong> Excel, ApE, FinchTV, Image Studio Lite, NEB, Epidata, Metamorph for Olympus, ImageJ, Genebank</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
